### PR TITLE
fix(docs): fix Dependency-injection link

### PR DESCRIPTION
**Pull request by byao**
## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/reference/reserved-binding-keys.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Reserved-binding-keys.html
 ---
 
-When using [dependency injection](Dependency-injection.md) there are a few
+When using [dependency injection](Dependency-injection.html) there are a few
```
Comment (dhmlau) [inline docs/site/reference/reserved-binding-keys.md:10]: we're trying to consistently use the .md file of the docs. Could you please update it accordingly? Thanks. 
Here is the corresponding md file: https://github.com/strongloop/loopback-next/blob/master/docs/site/Dependency-injection.md.

**Review by dhmlau — COMMENTED** [review]
@byao, thanks for your PR. Please see my comment of referencing the .md file instead. Thanks.

File: docs/site/reference/reserved-binding-keys.md (hunk 1 above)
Comment (achrinza) [reply docs/site/reference/reserved-binding-keys.md:10]: As what @dhmlau as mentioned, let's reference the markdown file directly.

```suggestion
When using [dependency injection](../Dependency-injection.md) there are a few
```

**Review by achrinza — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a minor change request.

**Review by raymondfeng — APPROVED** [review]

**Review by achrinza — APPROVED** [review]

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
@byao, your PR has landed! 🎉  Thanks!

**Pull request merged**